SSA/Projects/InstCombine/AliveHandwrittenLargeExamples.lean

```diff
@@ -134,6 +135,7 @@ def alive_simplifyMulDivRem805 (w : Nat) :
       rw [LLVM.sdiv?_denom_zero_eq_none]
       apply Refinement.none_left
     case neg =>
+      simp only [add_tsub_cancel_right, false_and, or_self, ↓reduceIte,Option.some_bind]
```

Comment:
This change, and others in this file, seem unrelated, in the sense that I can't see why the old proof might've broken

Reply:
I would have thought so as well, but it turns out that this line is nessecary.